### PR TITLE
[#172992621] Adds the available list screen and load the available bonuses

### PR DESCRIPTION
**Pull request by CrisTofani**
**Short description:**
This PR adds the Available bonuses list screen and displays the active bonus if a bonus was already activated

**List of changes proposed in this pull request:**
- ts/features/bonusVacanze/components/AvailableBonusItem.tsx
- ts/features/bonusVacanze/screens/AvailableBonusScreen.tsx
- ts/navigation/WalletNavigator.ts
- ts/screens/wallet/WalletHomeScreen.tsx
- ts/store/actions/navigation.ts

**How to test:**
Activate the flag `BONUS_VACANZE_ENABLED` on `.env` file then on wallet home screen click on `Request new bonus` button.

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -1238,6 +1238,9 @@ bonus:
   bonusVacanza:
     checkBonusEligibility:
       loading: Checking bonus eligibility
+  bonusList:
+    title: Bonus disponibili
```
Comment (Undermaken) [inline locales/en/index.yml:1242]: should be in english

File: ts/navigation/WalletNavigator.ts [hunk 2]
```diff
@@ -74,6 +75,11 @@ const WalletNavigator = createStackNavigator(
     },
     [ROUTES.PAYMENT_HISTORY_DETAIL_INFO]: {
       screen: PaymentHistoryDetailsScreen
+    },
+
+    // Navigator section for Bonus section
+    [ROUTES.AVAILABLE_BONUS_LIST]: {
```
Comment (Undermaken) [inline ts/navigation/WalletNavigator.ts:81]:
```suggestion
    [ROUTES.BONUS_AVAILABLE_BONUSES]: {
```
Perhaps having BONUS prefix for all bonus screens could be better when we trace them in Mixpanel

Comment (CrisTofani) [reply ts/navigation/WalletNavigator.ts:81]: Yep, sounds good! I'm changing it.

Comment (CrisTofani) [reply ts/navigation/WalletNavigator.ts:81]: Maybe just `BONUS_AVAILABLE_LIST` could be less redundant?

**Pull request merged**